framework/test/utils/node/state_store_mock.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
+import { hexToBuffer } from '@liskhq/lisk-cryptography';
+import { BlockHeader } from '@liskhq/lisk-chain';
+import { defaultAccount } from './account';
+
+interface Account<T = any> {
+	readonly address: Buffer;
+	balance: bigint;
+	nonce: bigint;
+	keys: {
+		mandatoryKeys: Buffer[];
+		optionalKeys: Buffer[];
+		numberOfSignatures: number;
+	};
+	asset: T;
+}
+
+export interface ChainState {
+	readonly lastBlockHeader: BlockHeader;
```

Comment:
i thought i changed the definition on the interface to be 
```
readonly lastBlockHeaders: BlockHeader[];
```
since dpos will require last X blocks to generate random seed